android/app/src/androidTest/java/org/fossasia/openevent/DateTest.java

```diff
@@ -0,0 +1,362 @@
+package org.fossasia.openevent;
+
+import org.fossasia.openevent.utils.ISO8601Date;
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * Created by arpitdec5 on 02-06-2017.
+ */
+
+public class DateTest {
+
+    @Test
+    public void shouldConvertSingaporeDateStringToUTCDateString() {
```

Comment:
1 testing 4 dates for correct conversion, 2nd for local to international testing only one string but on 4 timezones, 3rd one international to local 1 string for 4 TZ